api/constants/constants.go

```diff
@@ -332,11 +332,16 @@ const (
 	SessionRecordingServiceSSH SessionRecordingService = iota
 )
 
-// SessionRecordingMode determines how session recording will behave in failure
-// scenarios.
+// SessionRecordingMode determines how session recording will behave.
 type SessionRecordingMode string
 
 const (
+	// SessionRecordingModeUnspecified non-specified session recording mode.
+	SessionRecordingModeUnspecified = SessionRecordingMode("")
+
+	// SessionRecordingModeOff disables session recording.
+	SessionRecordingModeOff = SessionRecordingMode("off")
```

Comment:
This is to maintain consistency in returning/relying on the `SessionRecordingMode` "enum" where applicable. I agree that the off option here means the same, but I don't know if merging both sets of options or casting the values would make it more readable.

> We also have a slice over at types.SessionRecordingModes which has the same name as these constants but serves a different purpose.

Yeah, I've considered renaming, but I couldn't find a good alternative. From the user perspective, the option is called `record_session`. We might rename it to that, but it would still not be obvious. Anyway, I'll add a comment to the type that this is used on the role level.

Reply:
Maybe just add a role prefix?